Added a Collection of Pyodide Links (#438)
pyodide/pyodide: README.md

[![Build Status](https://circleci.com/gh/iodide-project/pyodide.png)](https://circleci.com/gh/iodide-project/pyodide)

> Pyodide is cPython compiled to WebAssembly and running in the browser.  It can run any pure Python package, and 

Reviewer: ```suggestion
Pyodide is CPython compiled to WebAssembly and running in the browser.  It can run any pure Python package, and 
```

> Pyodide is cPython compiled to WebAssembly and running in the browser.  It can run any pure Python package, and 

Reviewer: > It can run any pure Python package,

This isn't really true.  Anything with networking, such as Requests, would be right out, for example.

includes over 37 of Python's [best Data Science libraries](./packages) compiled to WebAssembly.

> Pyodide works great with Javascript. Proxy objects make it easy for Python and Javascript to interoperate. 

Reviewer: > Proxy objects make it easy for Python and Javascript to interoperate. 

It's really the "transparent conversion" that makes it easy to interoperate, of which Proxies are just one part.  I think we just leave the sentence about Proxies out here -- it's kind of an implementation detail.

Pyodide provides transparent conversion of objects between Javascript and Python.

> When inside a browser, this means Python has full access to the Window and Document objects.

Reviewer: The `window` and `document` objects are just a small part of the Web API that Pyodide has access to.  Why limit it to just those two?  If the concern that the phrase "Web API" isn't well-known, maybe link to: https://developer.mozilla.org/en-US/docs/Web/API

> If you are using an existing Python library, and want to run it in the browser, then Pyodide is the choice for you.  There are 4 commun use cases. 

Reviewer: ```suggestion
If you are using an existing Python library, and want to run it in the browser, then Pyodide is the choice for you.  There are 4 common use cases. 
```

> 1. You have limited Python compute server capacity. 

Reviewer: ```suggestion
1. The cloud computing resources would be cost-prohibitive.
```

> 2. It takes longer to updoad the data, then to download the application. 

Reviewer: ```suggestion
2. To move computation to the data, rather than the other way around.
```

3. You want the user interface to a Python application to be very responsive.

> 4. Medical privacy rules prevent your users from uploading data. 

Reviewer: ```suggestion
4. To preserve privacy by not moving data to another location
```

**While closely related to the [iodide project](https://iodide.io), Pyodide may
be used standalone in any context where you want to run Python inside a web
browser.**

For more information, see [the demo](https://extremely-alpha.iodide.io/notebooks/222/) and the
[documentation](https://github.com/iodide-project/pyodide/tree/master/docs).   Here is a 

> [collection of Pyodide Links](https://pythonlinks.info/pyodide).

Reviewer: These resources aren't part of the official documentation, so let's break this out into its own paragraph and say:

> [Your name here] maintains a [collection of Pyodide Links](https://pythonlinks.info/pyodide).
